asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -332,8 +401,15 @@ const ExportPage = (props) => {
           onClose={resetQueries}
         />
       )}
+
     </Root>
   );
 };
 
-export default ExportPage;
+const mapStateToProps = (state) => {
+  return {
+    asreview_version: state.asreview_version,
+  };
+};
+
+export default connect(mapStateToProps)(ExportPage);
```

Comment:
This code is here to pass the version number to the component. We are not using it now. I suggest inserting it into the citation styles. See the earlier version by @VeenDuco .

Reply:
I have moved the code to the new file CiteDialog